site/gatsby-site/gatsby-node.js

```diff
@@ -318,9 +318,12 @@ exports.onPreBuild = function ({ reporter }) {
 };
 
 exports.onPostBuild = () => {
-  // Replace Env variables on static file
+  console.log('Replacing Env variables on static file...');
```

Comment:
consider using `reporter.info()` here and below?

Reply:
Done